src/components/GoodCategory/GoodCategory.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {Good} from '../../api/api'
+import {CardItem} from '../Card'
+import css from './goodCategory.module.css'
+import {List } from 'antd';
+import { Divider } from 'antd';
+
+
+interface GoodCategory{
```

Comment:
```suggestion
interface GoodCategoryPorops {
```
GoodCategory - это компонент, а это пропсы. вообще он ругаться должен на одинаковые имена